src/main/java/org/terasology/blockNetwork/BlockNetworkUtil.java

```diff
@@ -40,6 +53,13 @@ private static Rotation getRotation(Block block) {
         return rotation;
     }
 
+    /**
+    * Give the new side of the block based on its' rotation
+    *
+    * @param block - the block that is about to be checked its' rotation and counted its' connections
+    * @param definedSides - the definedSides on which the loop iterates the sides of the block
+    * @return a byte of sides
```

Comment:
I'd like information on what the byte actually contains? A byte is a series of 1's and 0's. How does that relate to sides (this could point to another bit of javadoc)

Reply:
MichaelP said that each bit in the byte represents one side...